src/components/Sections/Library/LibraryContainer.js

```diff
@@ -363,6 +413,25 @@ class LibraryContainer extends Component {
                     <LibrarySettings fontSize={this.state.fontSize} handleSettings={this.handleSettings} />
                     <Button compact size="small" icon={isReadable ? 'compress' : 'expand'} onClick={this.handleIsReadable} />
                     <Button compact size="small" className="computer-hidden large-screen-hidden widescreen-hidden" icon="list layout" onClick={this.handleTocIsActive} />
+                    <Button compact size="small" icon="share alternate" onClick={this.handleShare} />
+                    {
+                      share ?
+                        <div className="share-bar">
+                          <ShareBar url={url} t={t} buttonSize="mini" messageTitle="see the article" />
+                          <Message content={url} size="mini" />
+                          <Popup
```

Comment:
@wissog I don't see the Popup (this is not css tweaks, it's UI functionallity - please implement fully)